src/event_loop.rs

```diff
@@ -125,23 +121,23 @@ impl EventSource for WaylandSource {
         &mut self,
         poll: &mut calloop::Poll,
         token_factory: &mut TokenFactory,
-    ) -> std::io::Result<()> {
+    ) -> Result<(), calloop::Error> {
```

Comment:
This should be io::Result?

Reply:
when I try to use `io::Result`, I see an error. `calloop::Result` is fine though